src/components/navigation/document-tab-panel.tsx

```diff
@@ -142,33 +144,56 @@ export class DocumentTabPanel extends BaseComponent<IProps, IState> {
   private renderSubSections(subTab: any) {
     const { selectedDocument, onSelectNewDocument, onSelectDocument } = this.props;
     const { user } = this.stores;
+    const isInNetwork = user.type === "teacher" && user.network;
+    const currentClass = this.stores.class.name;
+    const classNamesStrings = (uniq(user.portalClassOfferings.map(o => o.className))).filter(c => c !== currentClass);
```

Comment:
There's no guarantee that class names are unique, so it'd be better to filter on class hash or id.

Reply:
Same as above, I migrated this code from another component, but I might as well fix it now.